conf/sample.ini

```
@@ -386,6 +386,7 @@
 ;team_ids =
 ;allowed_organizations =
 ;role_attribute_path =
+;role_required = true
```

Should have the default value here shouldn't it:
```suggestion
;role_required = false
```